tools/docker/run.sh

```diff
@@ -61,7 +47,7 @@ main() {
             -h | --help)        usage; exit 0; shift ;;
             -d | --detach)      DETACH=true; shift ;;
             -f | --force)       FORCE=true; shift ;;
-            -i | --ipaddr)      IPADDR="$2"; shift; shift ;;
+            -i | --ipaddr)      IPADDR="--ip=$2"; shift; shift ;;
```

Comment:
I don't believe anyone uses this option anyway, so let's just remove it.

Reply:
Ok, done.